source/core_mqtt.c

```diff
@@ -2678,7 +3175,7 @@ MQTTStatus_t MQTT_CancelCallback( const MQTTContext_t * pContext,
 
 MQTTStatus_t MQTT_Connect( MQTTContext_t * pContext,
                            const MQTTConnectInfo_t * pConnectInfo,
-                           const MQTTPublishInfo_t * pWillInfo,
```

Comment:
I see const is removed because we are updating pWillInfo->propertyLength, why is this specific update is needed for v5?